6_temp_coop_fetch.yml

```diff
@@ -0,0 +1,45 @@
+target_default: 6_temp_coop_fetch
+
+include:
+
+packages:
+  - scipiper
+  - dplyr
+  - googledrive
+
+sources:
+  - 6_temp_coop_fetch/src/filter_files.R
+  - 6_temp_coop_fetch/src/task_fxns.R
+
+targets:
+
+  6_temp_coop_fetch:
+    depends:
+      - 6_temp_coop_fetch_tasks.yml
+
+  # -- prepare destination folders -- #
+  # in = folder to hold coop data files and corresponding .ind that need to be parsed (not on google drive)
+  # log = folder for indivator files that don't correspond to data file
+
+  6_temp_coop_fetch/in:
+    command: dir.create(target_name, showWarnings = I(FALSE))
+
+  6_temp_coop_fetch/log:
+    command: dir.create(target_name, showWarnings = I(FALSE))
+
+  # -- create makefile with targets for each file listed in gd -- #
+
+  coop_file_upload_location:
+    command: as_id(I('1dutCiFEOoRObXLn6n3BRKhZDDAijK65Q'))
+
+  coop_all_files:
+    command: drive_ls(path = coop_file_upload_location)
```

Comment:
>  I imagine that behavior you're seeing and not expecting is that coop_all_files never gets updated, no matter how often you call scmake()?

Correct - I added a test file to the google drive, and it won't rebuild unless I call `scmake('coop_all_files', force=TRUE)`. Is this an okay behavior, or is worth writing in something that tricks `scipiper` - e.g., adding the current date somehow, so it will at least rebuild `coop_all_files` on a daily basis. @jread-usgs -- do you know if MN peeps are still adding data on the regular? it looks from the GD that this is happening ~weekly.

Reply:
Yes, I think they will continue to add data fairly regularly, and then things will probably slow down with the start of the semester.

Reply:
I think it's an OK behavior. Your message prompted me to try a couple more approaches to trick scipiper/remake, but I just can't do it! I think the force=TRUE approach is just fine, maybe even preferable to rebuilding every time.